README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[x] Strings
+[x] Booleans
+[x] Undefined
 [] NaN
-[] Integers
-[] Arrays
-[] Null
+[x] Integers
+[x] Arrays
+[x] Null
```

Comment:
Actually, arrays are technically *not* a primitive data type: they're a type of object. But they're so fundamental that I'll give it to you. May be a good thing to keep in mind for anal-retentive job interviewers, though.